test/sanity/issue5781-X-Frame-Options/test.py

```diff
@@ -0,0 +1,28 @@
+import time
+import os
+import sys
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from nw_util import *
+
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+
+chrome_options = Options()
+testdir = os.path.dirname(os.path.abspath(__file__))
+chrome_options.add_argument("nwapp=" + testdir)
+
+driver = webdriver.Chrome(executable_path=os.environ['CHROMEDRIVER'], chrome_options=chrome_options)
+driver.implicitly_wait(5)
+time.sleep(1)
+try:
+    print driver.current_url
+    print 'waiting to load https://web.whatsapp.com/'
+    wait_for_element_tag(driver, "iframe")
+    assert(driver.find_element_by_tag_name("iframe") is not None)
+    driver.switch_to.frame(driver.find_element_by_tag_name("iframe"))
+    result = wait_for_element_class(driver, '_2BXqy')
```

Comment:
This depends on the design of remote site, which could be changed and break our case in future.

Reply:
Following we talked before, created a local http server instead of this remote site.